views/templates/front/mailalerts-account-line.tpl

```diff
@@ -23,15 +23,17 @@
 * International Registered Trademark & Property of PrestaShop SA
 *}
 
+<a href="#"
+   title="{l s='Remove mail alert' d='Modules.Mailalerts.Shop'}"
+   class="js-remove-email-alert btn btn-link"
+   rel="js-id-emailalerts-{$mailAlert.id_product|intval}-{$mailAlert.id_product_attribute|intval}"
+   data-url="{url entity='module' name='ps_emailalerts' controller='actions' params=['process' => 'remove']}"
+   style="float:right">
```

Comment:
I'm not satisfied with an inline style, but unfortunately bootstrap class `float-right` does not work. Does anyone have a suggestion?